tutorials/animation/cutout_animation.rst

```diff
@@ -103,52 +90,42 @@ Sprite:
+The pivot can also be adjusted *visually*. While hovering over the
+desired pivot point,  press the "v" key to move the pivot there for the
+selected Sprite. There is also a tool in the tool bar that has a
 similar function.
 
 .. image:: img/tutovec_torso2.gif
 
-Now it looks good! Let's continue adding body pieces, starting by the
-right arm. Make sure to put the sprites in hierarchy, so their rotations
-and translations are relative to the parent:
+Continue adding body pieces, starting with the
+right arm. Make sure to put each sprite in its correct place in the hierarchy, so its rotations
+and translations are relative to its parent:
 
 .. image:: img/tuto_cutout6.png
 
-This seems easy, so continue with the left arm. The rest should be
-simple! Or maybe not:
+With the left arm there's a problem. In 2D, child nodes appear in front of their parents:
 
 .. image:: img/tuto_cutout7.png
 
-Right. In 2D, parent nodes appear below children nodes. Well, this sucks.
-But how can this problem be solved? We want the left arm to appear behind
-the hip and the torso. For this, we can move the nodes behind the hip
-(note that you can bypass this by setting the Node2D Z property, but then you
-won't learn about all this!):
-
-.. image:: img/tuto_cutout8.png
+We want the left arm to appear *behind*
+the hip and the torso. We could move the left arm nodes behind the hip (above the hip node in the scene heirarchy), but then the left arm is no longer in its proper place in the heirarchy. This means it wouldn't be affected the movement of the torso. We'll fix this problem with `RemoteTransform2D` nodes.
 
-But then, we lose the hierarchy layout, which allows to control the
-skeleton like.. a skeleton. Is there any hope?.. Of Course!
+.. note:: You can also fix depth ordering problems by adjusting the Z property of any node inheriting from Node2D.
 
 RemoteTransform2D node
 ~~~~~~~~~~~~~~~~~~~~~~
 
-Godot provides a special node, :ref:`RemoteTransform2D <class_RemoteTransform2D>`.
-This node will transform nodes that are sitting somewhere else in the
-hierarchy, by applying the transform to the remote nodes.
+The :ref:`RemoteTransform2D <class_RemoteTransform2D>` node transforms nodes somewhere else in the
+hierarchy. This node applies its own transform (including any transformation it inherits from its parents) to the remote node it targets.
 
-This enables to have a visibility order independent from the
-hierarchy.
+This allows us to correct the visibility order of our elements independent from the
+locations of those parts in the cutout hierarchy.
 
-Simply create two more nodes as children from torso, remote_arm_l and
-remote_hand_l and link them to the actual sprites:
+Create a `RemoteTransform2D` node as a child of the torso. Call it remote_arm_l. Create another RemoteTransform2D node inside the first and call it remote_hand_l. Use the Remote Path property of the two new nodes to target the arm_l and hand_l sprites respectively:
```

Comment:
Note that you should use double backticks for inline code in .rst, i.e.
<pre>
``RemoteTransform2D``
</pre>
Please also use inline code markup for any variable/node name like `remote_arm_l`.